articles/quickstart/webapp/nodejs/01-login.md

```diff
@@ -12,365 +12,103 @@ topics:
-:::
+A logout URL is an application route that Auth0 can return users to after logging out. This URL must be registered with Auth0 or else users will be unable to log out of the application and will get a "misconfiguration" error.
 
-If you are using git, create a `.gitignore` file (or edit your existing one, if you have one already) and add `.env` to it. The `.gitignore` file tells source control to ignore the files (or file patterns) you list. Be careful to add `.env` to your `.gitignore` file and commit that change before you add your `.env`.
+The logout URL for the application created in this quickstart is `http://localhost:3000`. Paste that in the **Allowed Logout URLs** field for the application you just created, then scroll down and click **Save Changes**.
 
-```
-# .gitignore
-.env
-```
+### 3. Get your application keys
 
-### Install the dependencies
+Finally, copy the following fields for your application for use in step 7:
 
-To get started, install the following dependencies.
+* **Domain**
+* **Client ID**
 
-* [passport](http://www.passportjs.org/) - an authentication middleware for Node.js
-* [passport-auth0](https://github.com/auth0/passport-auth0) - an Auth0 authentication strategy for Passport
-* [express-session](https://www.npmjs.com/package/express-session) - a middleware to manage sessions
-* [dotenv](https://www.npmjs.com/package/dotenv) - a module to load environment variables from a `.env` file
+## Integrate Auth0
+### 4. Install dependencies
+Your application will need the [`express-openid-connect`](https://github.com/auth0/express-openid-connect) package, an Auth0-maintained OIDC-compliant library for Express.
 
-```bash
-# installation with npm
-npm install passport passport-auth0 express-session dotenv --save
+```sh
+npm install express express-openid-connect --save
 ```
 
-### Configure express-session
-
-In `app.js`, include the `express-session` module and configure it. The `secret` parameter is a secret string that is used to sign the session ID cookie. Please use a custom value.
-
-```js
-// app.js
-
-var session = require('express-session');
-
-// config express-session
-var sess = {
-  secret: 'CHANGE THIS TO A RANDOM SECRET',
-  cookie: {},
-  resave: false,
-  saveUninitialized: true
-};
-
-if (app.get('env') === 'production') {
-  // Use secure cookies in production (requires SSL/TLS)
-  sess.cookie.secure = true;
+### 5. Configure router
+The Express OpenID Connect library provides the `auth` router in order to attach authentication routes to your application. You will need to configure the router with the following configuration keys:
 
-  // Uncomment the line below if your application is behind a proxy (like on Heroku)
-  // or if you're encountering the error message:
-  // "Unable to verify authorization request state"
-  // app.set('trust proxy', 1);
-}
+- `baseURL` - The URL where the application is served
+- `appSessionSecret` - A long, random string
```

Comment:
Should this be now renamed to `appSession.secret` or something else to reflect the change?